python/gps/agent/mjc/agent_mjc.py

```diff
@@ -132,15 +149,37 @@ def _init_sample(self, condition):
                    self._hyperparams['x0'][condition][self._joint_idx], t=0)
         sample.set(JOINT_VELOCITIES,
                    self._hyperparams['x0'][condition][self._vel_idx], t=0)
-        self._data = self._world.get_data()
+        self._data = self._world[condition].get_data()
         eepts = self._data['site_xpos'].flatten()
         sample.set(END_EFFECTOR_POINTS, eepts, t=0)
         sample.set(END_EFFECTOR_POINT_VELOCITIES, np.zeros_like(eepts), t=0)
         jac = np.zeros([eepts.shape[0], self._model[condition]['nq']])
         for site in range(eepts.shape[0] // 3):
             idx = site * 3
-            jac[idx:(idx+3), :] = self._world.get_jac_site(site)
+            jac[idx:(idx+3), :] = self._world[condition].get_jac_site(site)
         sample.set(END_EFFECTOR_POINT_JACOBIANS, jac, t=0)
+
+        # save initial image to meta data
+        self._world[condition].plot(self._hyperparams['x0'][condition])
+        img = self._world[condition].get_image_scaled(self._hyperparams['image_width'],
+                                                      self._hyperparams['image_height'])
+        # mjcpy image shape is [height, width, channels], dim-shuffle it for later conv-net processing,
```

Comment:
nit pic: can you shorten this line? I think we're generally aiming for <=80 characters per line